pkg/controller/provider/web/vsphere/network.go

```diff
@@ -188,7 +188,7 @@ func (r *Network) With(m *model.Network) {
 	switch m.Variant {
 	case model.NetStandard:
 		r.Tag = m.Tag
-	case model.NetDvPortGroup:
+	case model.NetDvPortGroup, model.OpaqueNetwork:
 		r.DVSwitch = &m.DVSwitch
```

Comment:
`model.OpaqueNetwork` should probably have its own case since it won't have a value for DVSwitch.

Reply:
Updated, thanks!